components/director/internal/domain/labeldef/entity.go

```diff
@@ -0,0 +1,8 @@
+package labeldef
+
+type Entity struct {
+	ID         string `db:"id"`
+	TenantID   string `db:"tenantID"`
+	Key        string `db:"key"`
+	SchemaJSON string `db:"schemaJSON"`
```

Comment:
if SchemaJSON is a `string` and user does not specify the schema while creating new Label Definition it will be an empty string which won't be a correct value for `jsonb` so db will reject the creation.

Reply:
schemaJSON is now sql.NullString, it is saved only if it is provided.